scalafix-tests/input/src/main/scala/test/NoExtendsApp.scala

```diff
@@ -0,0 +1,46 @@
+/*
+rewrites = [NoExtendsApp]
+*/
+package test
+
+object Main extends App {
+  println(s"Hello, ${args(0)}")
+  println(s"Hello 2, ${args(0)}")
+}
+
+trait Something
+object Main2 extends Something with App{
+  println(s"Hello, ${args(0)}")
+  println(s"Hello 2, ${args(0)}")
+}
+
+object Main3 extends App with Something{
+  println(s"Hello, ${args(0)}")
+  println(s"Hello 2, ${args(0)}")
+}
+
+object Main4 extends App with Something
+
+object Main5 extends App with Something {}
+
+object Main6
+  extends App
+  with Something
+
+object Main7
+  extends Something
+  with App
+
+object Main8
+  extends Something with
+  App
+
+object Main9 extends
```

Comment:
Hehe, token level rewrites are hard. I wish I could come up with a more intuitive or higher level API for common tasks